src/main/java/org/osiam/client/AuthService.java

```diff
@@ -50,13 +62,18 @@
  */
 class AuthService {
 
+    private static final String AUTHORIZATION_HEADER = "Authorization";
     private static final String BEARER = "Bearer ";
     private static final String TOKEN_ENDPOINT = "/oauth/token";
+    private static final String TOKEN_REVOCATION_ENDPOINT = "/token/revocation";
+    private static final String TOKEN_VALIDATION_ENDPOINT = "/token/validation";
 
     private final String endpoint;
     private final String clientId;
     private final String clientSecret;
     private final String clientRedirectUri;
+    private final int connectionTimeout;
+    private final int readTimeout;
```

Comment:
Do you really think this is necessary, because this isn't part of the Public API and it's only [used once](https://github.com/osiam/connector4java/pull/167/files#diff-e61d51b09a914e6b31732b23b8dc2161R195) and [always set with a value](https://github.com/osiam/connector4java/pull/167/files#diff-e61d51b09a914e6b31732b23b8dc2161R180)